constraint_utils/constraint_basics.py

```diff
@@ -227,6 +247,15 @@ def filter_to_autosomes_par(ht: Union[hl.Table, hl.MatrixTable]) -> Union[hl.Tab
 
 def annotate_with_mu(ht: hl.Table, mutation_ht: hl.Table, output_loc: str = 'mu_snp',
                      keys: Tuple[str] = ('context', 'ref', 'alt', 'methylation_level')) -> hl.Table:
+    """Annotate SNP mutation rate for the input Table.
+
+    :param ht: Input table
+    :param mutation_ht: Mutation rate table
+    :param output_loc: Name for mutational rate annotation. Defaults to 'mu_snp'.
+    :param keys: Keys of output table. Defaults to ('context', 'ref', 'alt', 'methylation_level').
```

Comment:
Is this the keys of the output HT? It looks like it's using `keys` to map the `mu_snp` from the `mutation_ht` to the `ht`. I don't see any sort of rekey, so I think the output HT will have as a key whatever it had coming in